package.json

```diff
@@ -18,7 +18,7 @@
   },
   "homepage": "https://github.com/dumbledorep/Phil-s-Portfolio#readme",
   "devDependencies": {
-    "hint": "^7.0.1",
+    "hint": "^7.1.0",
     "stylelint": "^13.13.1",
     "stylelint-config-standard": "^21.0.0",
```

Comment:
kindly make sure your page is responsive and match the design. You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)